cavern/build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
     compile 'org.opencadc:cadc-cdp:[1.0,)'
     compile 'org.opencadc:cadc-gms:[1.0.0,)'
     compile 'org.opencadc:cadc-vos:[1.2.3,)'
-    compile 'org.opencadc:cadc-vos-server:[1.3,)'
+    compile 'org.opencadc:cadc-vos-server:[1.3.4,)'
```

Comment:
would prefer to not have any changes in cavern, even cosmetic ones

Reply:
also thew change in FilesAction should be reverted

Reply:
Done